bin/check-commit-msg.php

```diff
@@ -112,6 +113,12 @@ function is_not_comment( $line ) {
 	'The commit message must end with a full stop.'
 );
 
+// single word commit.
+$error_code |= echo_error_if(
+	count( $words ) < 2,
+	'The commit message cannot be a signle word.'
```

Comment:
Typo 😄  - also, let's use a similar message form stating what the message must be rather than what it cannot be.
```suggestion
	'The commit message must be more than one word.'
```